sceptre/securitycentral/templates/sechub_suppress_infra.yaml

```diff
@@ -0,0 +1,273 @@
+        - Action: kms:Decrypt
+          Effect: Allow
+          Principal:
+            AWS:
+              Fn::GetAtt:
+              - SecurityHubBatchUpdateSuppressionRole
+              - Arn
+          Resource: "*"
+        - Action:
+          - kms:Decrypt
+          - kms:Encrypt
+          - kms:ReEncrypt*
+          - kms:GenerateDataKey*
+          Effect: Allow
+          Principal:
+            Service: events.amazonaws.com
+          Resource: "*"
+        Version: '2012-10-17'
+      EnableKeyRotation: true
+      PendingWindowInDays: 30
+    UpdateReplacePolicy: Retain
+    DeletionPolicy: Retain
+  SqsEncryptionKeyAlias:
+    Type: AWS::KMS::Alias
+    Properties:
+      AliasName: alias/SQSEncryptionKey
+      TargetKeyId:
+        Fn::GetAtt:
+        - SqsEncryptionKey
+        - Arn
+  SecurityhubFindingsDeadLetterQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      KmsMasterKeyId:
+        Fn::GetAtt:
+        - SqsEncryptionKey
+        - Arn
+      MessageRetentionPeriod: 172800
+      VisibilityTimeout: 130
+    UpdateReplacePolicy: Delete
+    DeletionPolicy: Delete
+  SecurityHubFindingsDeadLetterQueuePolicy:
+    Type: AWS::SQS::QueuePolicy
+    Properties:
+      PolicyDocument:
+        Statement:
+        - Action: sqs:*
+          Condition:
+            Bool:
+              aws:secureTransport: 'false'
+          Effect: Deny
+          Principal: "*"
+          Resource:
+            Fn::GetAtt:
+            - SecurityhubFindingsDeadLetterQueue
+            - Arn
+          Sid: Enforce TLS for all principals
+        Version: '2012-10-17'
+      Queues:
+      - Ref: SecurityhubFindingsDeadLetterQueue
+  SecurityHubFindingsQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      KmsMasterKeyId:
+        Fn::GetAtt:
+        - SqsEncryptionKey
+        - Arn
+      RedrivePolicy:
+        deadLetterTargetArn:
+          Fn::GetAtt:
+          - SecurityhubFindingsDeadLetterQueue
+          - Arn
+        maxReceiveCount: 1
+      VisibilityTimeout: 130
+    UpdateReplacePolicy: Delete
+    DeletionPolicy: Delete
+  SecurityHubFindingsQueuePolicy:
+    Type: AWS::SQS::QueuePolicy
+    Properties:
+      PolicyDocument:
+        Statement:
+        - Action: sqs:*
+          Condition:
+            Bool:
+              aws:secureTransport: 'false'
+          Effect: Deny
+          Principal: "*"
+          Resource:
+            Fn::GetAtt:
+            - SecurityHubFindingsQueue
+            - Arn
+          Sid: Enforce TLS for all principals
+        - Action:
+          - sqs:SendMessage
+          - sqs:GetQueueAttributes
+          - sqs:GetQueueUrl
+          Effect: Allow
+          Principal:
+            Service: events.amazonaws.com
+          Resource:
+            Fn::GetAtt:
+            - SecurityHubFindingsQueue
+            - Arn
+        Version: '2012-10-17'
+      Queues:
+      - Ref: SecurityHubFindingsQueue
+
+  # In every account, suppress findings for CIS4.3 and CIS1.14
```

Comment:
is there a web page that provides more info about these CIS findings? if so could you please add that link in here or in the README file?